### DISCUSSION TITLE
SCRUM PREP milestone 2-3

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#227] [General]
Held scrum meetings, had weekly team meeting, closed PR and reviewed issues

**Comment by nsfrontera** [comment]
Worked on my tasks, still have more work to do with them.

**Comment by ianramospr** [comment]
Finished the game overlays for when a player wins or loses, and when it's their turn to play.

**Comment by RicardoJRSs** [comment]
Worked on my tasks and fixed documentation

**Comment by sandy-marrero** [comment]
- Started work on #195 
- Looking into #173

**Comment by DavidCuevas3** [comment]
Finished the task for the first sprint of milestone 2. Said task being changing the multiplayer leaderboard to better match the wooden and pixelated aesthetic of the game. #220

**Comment by AngelCIICMorales** [comment]
Held weekly meetings, reviewed tasks, and bug tested the project.

**Comment by Saul-Caban-UPRM** [comment]
Created task #226 and started to work on that task

**Comment by fernandopizarro1** [comment]
Continued with the implementation of the AdvancedBot, also looking into the position bug and an error with the bot, where it is not updating accordingly.

**Comment by PinkSylvie** [comment]
Continued reading up on aggregates and invariants to start writing that part of the documentation hopefully this weekend

**Comment by ebrain-mendoza121** [comment]
Attended meetings, worked on lobby task and fixed verification and validation task.

**Comment by FrancesC8** [comment]
Started work on refactoring tasks to adopt SOLID principles in backend #147 & #145

**Comment by JadiellMV** [comment]
This week I started working on the implementation of the password reset feature.

**Comment by AnthonyMed13** [comment]
Continued working on issue #146 which was to solve the JWT secret key problem

**Comment by Negroni1** [comment]
Added domain diagram to the documentation and review documentation to start implementing the professor's feedback on the interface and domain requirements and the scope, span, and the synopsis. Also helped the managers with the script that evaluates the students scores to ensure sprint 1 completion tasks

**Comment by rukitoyordan** [comment]
Provided two new boards that meet palette standards. During this weekend I will also be creating more simpler boards for evaluation and possibly more powerups for avatars. Working on two new LTTs alongside @frances-sepulveda.

**Comment by ilikecheese117** [comment]
Closed a task and did a pull request this week

**Comment by Gaby-Concepcion** [comment]
Finished adding new features into user stories and epics, while continuing to work on revamping the narrative of the documentation. I have planned to finish adding factories and a new narrative for the documentation by this weekend and then focusing the remainder of the milestone on finishing the coding implementations.

**Comment by frances-sepulveda** [comment]
Worked with new boards to match the game's palettes. Additionally, I am working on 2 LTT with @rukitoyordan.

**Comment by print-Eruki** [comment]
Work on Machine requirements following the guidelines provided for milestone 1

**Comment by G4vp** [comment]
Worked on creating Lecture Topic Tasks and began researching accessibility for users with disabilities.

**Comment by Jvstinoo** [comment]
Started work on implementing additional unit tests for correct game scenarios.
